protocol/triple/internal/client/cmd/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"context"
+	"dubbo.apache.org/dubbo-go/v3/client"
+	_ "dubbo.apache.org/dubbo-go/v3/imports"
+	greet "dubbo.apache.org/dubbo-go/v3/protocol/triple/internal/proto"
+	"dubbo.apache.org/dubbo-go/v3/protocol/triple/internal/proto/greettriple"
+	"github.com/dubbogo/gost/log/logger"
+)
+
+func main() {
+	//client initialization
+	cli, err := client.NewClient(
```

Comment:
Is `dubbogo.yml` still needed to run with this API?

Reply:
No, dubbogo.yml is only needed for classic configuration (SetConsumerService, config.Load). In this case, just create client with options.